import/dproto/attributes.d

```diff
@@ -40,6 +42,8 @@ template TagId(alias T)
 template ProtoAccessors()
 {
 
+@safe:
+
```

Comment:
OK. Didn't know about attribute inference :)
But still the templated `deserialize()` has to be marked as @safe too. Is it appropriate to require that the ProtoInputRange R provides @safe functions? My answer is yes, you should definitively enforce bound checks when parsing data from non-trusted sources.